tests/api2/test_pool_dataset_user_props.py

```diff
@@ -0,0 +1,28 @@
+import pytest
+
+from middlewared.test.integration.assets.pool import dataset
+from middlewared.test.integration.utils import call
+
+
+def test_pool_dataset_query_default_user_props():
```

Comment:
This test is unnecessary since you're validating same logic in the test below this.